eng/Generate.ps1

```diff
@@ -36,19 +27,24 @@ $debugFlags = if (-not $noDebug) { '--debug', '--verbose' }
 $testServerDirectory = Join-Path $repoRoot 'test' 'TestServerProjects'
 $configurationPath = Join-Path $testServerDirectory 'readme.tests.md'
 $testServerSwaggerPath = Join-Path $repoRoot 'node_modules' '@microsoft.azure' 'autorest.testserver' 'swagger'
-$testNames = if ($name) { $name } else { 'extensible-enums-swagger', 'url-multi-collectionFormat', 'url', 'body-string', 'body-complex', 'custom-baseUrl', 'custom-baseUrl-more-options', 'header' }
-
-if (!$NoReset)
+$testNames = if ($name) { $name } else
 {
-    Invoke-AutoRest "--reset" $repoRoot
+    'body-string',
+    'extensible-enums-swagger',
+    'url-multi-collectionFormat',
+    'url',
+    'body-complex',
+    'custom-baseUrl',
+    'custom-baseUrl-more-options',
+    'header'
```

Comment:
Made the test list vertical instead of horizontal since we'll be adding a lot more here in the near term (avoid having a really wide line).